f5router/f5router.go

```diff
@@ -186,7 +189,7 @@ func NewF5Router(
 ) (*F5Router, error) {
 	r := F5Router{
 		c:                    c,
-		logger:               logger,
+		logger:               logger.Session("f5router"),
```

Comment:
This is being set in the call to this function

Reply:
Ya I backed this change out